### DISCUSSION TITLE
fetch data in background when search params change

### DISCUSSION THREAD

**Discussion by pke** [paoloricciuti/sveltekit-search-params#14] [Q&A]
Say I have a certain functionality that can only be run on the server. This works fine on initial render (SSR).
However I'd like to call the same server API when a search param is changed.
I think that could work with the `load` function in `+page.server.ts` but it does not react upon search params change.

**Comment by paoloricciuti** [comment]
Pay attention to use stores on the server because they gets created on the server instance (meaning every client would get the same value)...if you can provide a minimal repro I can try to take a look tho

**Reply by pke** [reply]
Sure. I have modified the example

https://github.com/paoloricciuti/sveltekit-search-params/compare/master...pke:sveltekit-search-params:master

or in stackblitz:
https://stackblitz.com/~/github.com/pke/sveltekit-search-params

to sumarise: when the input length is modified, the query param should be updated and in the background the server function (API) should be called again and with its result the page data is modifed.
